### PR TITLE
Add Performance Support page to Solutions

### LINKED ISSUE #1758: Add Performance Support
We made a new solutions page that can be added on the website under Lösungen --> Performance Support with the URL https://www.defactolearning.de/performancesupport

Text can be found here: https://defactosoftwarebv.sharepoint.com/:w:/r/_layouts/15/Doc.aspx?sourcedoc=%7B4aac92ef-aa7d-4399-a8f1-1bdef1b29f8a%7D&action=edit&wdPid=69e3ac81&cid=c1cf3d25-c6eb-45c0-abb6-92944183dafb

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Close #1758 

- [x] Add page
 - [x] Add hero with background @RonaldDijkstra 
 - [x] Add copy @lennartdefacto (done by @RonaldDijkstra )
 - [x] Add images @lennartdefacto 
 - [x] Add meta description @lennartdefacto 
 - [ ] ~Add social image (if applicable) @lennartdefacto~
- [x] Add to main navigation (includes icon)  @RonaldDijkstra 
- [x] Add to solutions index @RonaldDijkstra 
- [ ] ~Add solutions to Footer (?) @RonaldDijkstra~ not here

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
I guess I am done (added all textual content too). See OP for your todos @lennartdefacto. There's a video in the input, some sort of screencast, I don't know what is up with that. We have a youtube channel, maybe publish it there and embed it on the page?

**Comment by lennartdefacto** [comment]
Published the video on our vimeo account with public viewing enabled, added the relevant images to their proper partials. Social images does not seem applicable in this instance? Same for meta description? If I'm wrong, where would I put it?

**Comment by RonaldDijkstra** [comment]
Description is relevant and should be in frontmatter.
Image can be default..

**Comment by isabelamy** [comment]
A few remarks:

1. images are blurry. can we get a better resolution.
2. image 'Vorteile': is it possible to blend it more into the page instead of making it look like a picture?
3. at the end of the section about 5 moments of need, there is a call to action in the text but no conversion. I suggested the demo bar but then with 'Termin vereinbaren' is that not possible?
4. qr code picture: can we turn it around and make the phone with the qr code bigger, and maybe make some sort of shadow or flash from the phone to the website.
5. video: i'm gonna look into whether we can make a gif from the video. i think that's nicer.

**Comment by lennartdefacto** [comment]
1. All the images? The performance one seems okay sharp-wise?
2. Yes, I can remove the white background and the dropshadow
3. @Ronald, can you do this easily?
4. Yes, can be done
5. I hope the gif won't be to big in size.

**Comment by ronald** [comment]
@lennartdefacto -> @RonaldDijkstra != @ronald

**Comment by RonaldDijkstra** [comment]
> @RonaldDijkstra , can you do this easily? 

Yes but two different conversions on a single page is a bad practice...

**Comment by RonaldDijkstra** [comment]
> All the images? The performance one seems okay sharp-wise? 

There is some work here to be done: make the images responsive. We don't do anything for that right now!

**Comment by lennartdefacto** [comment]
Isa has already told me she approves the new images, visually all thats left is for Isa to define what she wants with the GIF.

**Comment by RonaldDijkstra** [comment]
Good job Lennart

**Comment by RonaldDijkstra** [comment]
Since I committed to this branch we need another reviewer.

**Comment by lennartdefacto** [comment]
- ./build/blog/de-geschiedenis-van-onze-stadsvilla/index.html
  *  External link http://www.bemedia.nl/ failed: 500 No error

**Comment by RonaldDijkstra** [comment]
That BE Media link is removed from the master branch I think

**Comment by lennartdefacto** [comment]
@sn3p @EasterPeanut could you review this today, if you're working today that is? Would love to put this page online a.s.a.p. Thanks!

**Comment by RonaldDijkstra** [comment]
> That BE Media link is removed from the master branch I think 

Rebased this

File: source/stylesheets/pages/_performance-support.scss [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    h1 {
+      font-size: $base-font-size * 1.6;
+      margin: 0 auto;
+      max-width: 960px;
+
+      @include media($medium-up) {
+        font-size: $base-font-size * 2.4;
+      }
+    }
+  }
```
Comment (EasterPeanut) [inline source/stylesheets/pages/_performance-support.scss]: @RonaldDijkstra this CSS for the hero (and on-page-navigation) is repeated, can't a part of it be included in the hero partial?

Comment (RonaldDijkstra) [reply source/stylesheets/pages/_performance-support.scss]: Yes we should refactor product pages. 

#1762

**Pull request merged**